src/tests/common/tests-common.cpp

```diff
@@ -1,19 +1,19 @@
 #include "../catch.hpp"
 
-#include "../../common/Circle.hpp"
+#include "../../common/CircleRigidBody.hpp"
 
-SCENARIO( "Circles move correctly", "[circle]" ) {
+SCENARIO( "CircleRigidBody step correctly", "[circle]" ) {
   sf::Vector2f position = sf::Vector2f(0.f, 0.f);
-  Circle* circle = new Circle(0.f, position, sf::Color(0, 255, 0));
+  CircleRigidBody* circle = new CircleRigidBody(0.f, position);
   GIVEN( "a velocity" ) {
     circle->setVelocity(sf::Vector2f(10.f, 10.f));
-    WHEN( "move is called" ) {
-      circle->move();
+    WHEN( "step is called" ) {
+      circle->step();
       REQUIRE(circle->getPosition() == sf::Vector2f(10.f, 10.f));
     }
-    WHEN( "move is called many times" ) {
+    WHEN( "step is called many times" ) {
       for (int i=0; i < 10; i++) {
-        circle->move();
+        circle->step();
       }
       REQUIRE(circle->getPosition() == sf::Vector2f(10*10.f, 10*10.f));
```

Comment:
can you update these tests to check the rigid bodies locations? I think it's time we require tests in our CRs :(

Reply:
Not ready to write tests yet :(

Reply:
what do you mean